src/main/java/frc/robot/commands/shooter/ShootSpeaker.java

```diff
@@ -32,6 +32,19 @@ public boolean isFinished() {
         return (timer.hasElapsed(Constants.SHOOTER_TIME_AFTER_TRIGGER)) && (activated);
     }
 
+    @Override
+    public void execute() {
+        if (((RobotContainer.isRedAlliance() == true) && (((((drivetrain.getGyroAngle().getDegrees()) % 360) + 360) % 360) > 180)) || ((RobotContainer.isRedAlliance() == false) && (((((drivetrain.getGyroAngle().getDegrees()) % 360) + 360) % 360) < 180))) {
+            shooter.setShooterMotorRightSpeed(Constants.SHOOTER_MOTOR_LEFT_SPEED);
+            shooter.setShooterMotorLeftSpeed(Constants.SHOOTER_MOTOR_RIGHT_SPEED);
+        }
+        if (((RobotContainer.isRedAlliance() == false) && (((((drivetrain.getGyroAngle().getDegrees()) % 360) + 360) % 360) > 180)) || ((RobotContainer.isRedAlliance() == true) && (((((drivetrain.getGyroAngle().getDegrees()) % 360) + 360) % 360) < 180))) {
```

Comment:
What if the angle is exactly 180?

Reply:
Hint - this can be simplified with an `else` clause

Reply:
Done